django_project/frontend/src/containers/MainPage/Data/index.tsx

```diff
@@ -137,10 +137,10 @@ const DataList = () => {
                             const cellKeys = Object.keys(cellData[0]);
                             const generatedColumns = cellKeys.map((key) => ({
                                 field: key,
-                                headerName: key.split('_')
+                                headerName: key.replace("owned_species__", "").replace("taxon", "").replace("varbatim", "").split('_')
```

Comment:
can you change to serializer instead in the backend, so it will get a proper title